GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -15,6 +15,81 @@
+
+/// Starts an interactive verify flow on iOS.
+///
+/// The completion will be called at the end of this process.  Any saved verify state will be
+/// replaced by the result of this flow.
+///
+/// @param accountDetails A list of verifiable account details.
+/// @param presentingViewController The view controller used to present `SFSafariViewController` on
+///     iOS 9 and 10 and to supply `presentationContextProvider` for `ASWebAuthenticationSession` on
+///     iOS 13+.
+/// @param completion The optional block that is called on completion.  This block will
+///     be called asynchronously on the main queue.
+- (void)verifyAccountDetails:(NSArray<GIDVerifiableAccountDetail *> *)accountDetails 
+         presentingViewController:(UIViewController *)presentingViewController 
+                       completion:(nullable void (^)(GIDVerifiedAccountDetailResult *_Nullable verifyResult, 
+                                                   NSError *_Nullable error))completion;
+
+/// Starts an interactive verify flow on iOS using the provided hint.
+///
+/// The completion will be called at the end of this process.  Any saved verify state will be
+/// replaced by the result of this flow.
+///
+/// @param accountDetails A list of verifiable account details.
+/// @param presentingViewController The view controller used to present `SFSafariViewController` on
+///     iOS 9 and 10 and to supply `presentationContextProvider` for `ASWebAuthenticationSession` on
+///     iOS 13+.
+/// @param hint An optional hint for the authorization server, for example the user's ID or email
+///     address, to be prefilled if possible.
+/// @param completion The optional block that is called on completion.  This block will
+///     be called asynchronously on the main queue.
+- (void)verifyAccountDetails:(NSArray<GIDVerifiableAccountDetail *> *)accountDetails 
+         presentingViewController:(UIViewController *)presentingViewController 
+                             hint:(nullable NSString *)hint
+                       completion:(nullable void (^)(GIDVerifiedAccountDetailResult *_Nullable verifyResult, 
+                                                   NSError *_Nullable error))completion;
+
+/// Starts an interactive verify flow on iOS using the provided hint and additional scopes.
+///
+/// The completion will be called at the end of this process.  Any saved verify state will be
+/// replaced by the result of this flow. 
+///
+/// @param accountDetails A list of verifiable account details.
+/// @param presentingViewController The view controller used to present `SFSafariViewController` on
+///     iOS 9 and 10.
+/// @param hint An optional hint for the authorization server, for example the user's ID or email
+///     address, to be prefilled if possible.
+/// @param additionalScopes An optional array of scopes to request in addition to the basic profile scopes.
+/// @param completion The optional block that is called on completion.  This block will
```

Comment:
Same comment about simplifying this doc.

Reply:
Done.